spec/mailers/previews/candidate_mailer_preview.rb

```diff
@@ -561,10 +533,6 @@ def feedback_received_for_application_rejected_by_default_apply_again(reasons_tr
     CandidateMailer.feedback_received_for_application_rejected_by_default(application_choice, show_apply_again_guidance)
   end
 
-  def feedback_received_for_application_rejected_by_default_apply_again_redesigned_reasons
-    feedback_received_for_application_rejected_by_default_apply_again(:with_redesigned_rejection_reasons)
-  end
-
```

Comment:
could we also remove `reasons_for_rejection_with_qualifications` and `reasons_type` methods?